modules/messages/client/api/messages.api.js

```diff
@@ -0,0 +1,20 @@
+import axios from 'axios';
+import parseLinkheader from 'parse-link-header';
+
+export async function fetchThreads(params = {}) {
+  const { data: threads, headers } = await axios.get('/api/messages', { params: { limit: 2, ...params } });
+  let nextParams;
+  if (headers.link) {
```

Comment:
This might eventually be good thing to abstract as a "middleware" type function we can re-use for other paginated endpoints. Good as it is now of course.

Reply:
Yes, I agree! When the time comes.